src/mailadm/data/sysconfig/mailadm-prune.service

```
@@ -3,7 +3,8 @@ Description=Mailadm account pruning job
 After=network.target
 
 [Service]
-User=mailadm 
+User={mailadm_user}
```

```suggestion
User=vmail
Group=vmail
```

Apparently this works :)